portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,27 @@ function addRandomStock() {
   const stockContainer = document.getElementById('stock-container');
   stockContainer.innerText = stock;
 }
+
+function getGreetingAndJsonUsingArrowFunctions() {
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+  });
+}
+
+//Fetches comments from the server and adds them to the DOM.
```

Comment:
Small nit, there should always be a space after `//`

Reply:
Addressed